lib/classes/Swift/Plugins/DefaultSenderPlugin.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/*
+ * This file is part of SwiftMailer.
+ * (c) 2009 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Sets the sender of a message if none is set
+ *
+ * @author     Adam Zielinski
+ */
+class Swift_Plugins_DefaultSenderPlugin implements Swift_Events_SendListener
+{
+
+    /**
+     * The default sender email.
+     *
+     * @var String
+     */
+    private $_defaultSenderEmail;
```

Comment:
Afaik, Swiftmailer doesn't use underscore-prefixed names for private properties.